src/controllers/routers/staffs.js

```diff
@@ -0,0 +1,40 @@
+/*!
+ * OneSG API Server by TL Bootcamp#6 OneSG Team
+ * Copyright(c) 2020 TechLadies
+ * MIT Licensed
+ */
+
+'use strict';
+
+const url = require('url');
+const querystring = require('querystring');
```

Comment:
express has a method to check query parameters, we don't need another package to do so
pls remove the package from package.json (if it was saved) and remove this require here.

Reply:
changed to use req.query